jest.config.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
             // TODO: in the future, the following values should be increase to at least 85%
             statements: 83.4,
             branches: 70.1,
-            functions: 76.2,
+            functions: 76.18,
```

Comment:
We should work towards increasing test coverage and not decreasing it. Is there a particular reason for it?

Reply:
As discussed with @krusche, this is an exceptional case to reduce our workload.
We will rework the code soon, the implemented test cases would then be removed again.